setup.py

```diff
@@ -57,7 +61,9 @@ def get_environment_marker_support_level():
 
 
 def main():
-    extras_require = {}
+    extras_require = {
+        "dev": ["gitpython", "invoke", "pre-commit", "towncrier", "tox", "wheel"]
```

Comment:
we should switch to a setuptools with full environment marker support and

Reply:
Not sure I follow, can you clarify or point me to some resources?

Reply:
https://www.python.org/dev/peps/pep-0508/#environment-markers

allows requirements like `wheel; extra == 'test'`

Reply:
its also what will land in the wheels

Reply:
Hmm OK, but I don't get why this is not working right now when it seems it works in tox...